documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,169 @@
+Those issues require a different approach which is not covered in the scope of this document.
+```
+
+It is crucial to be proactive here; if something _might_ be an incident, but
+you’re not certain, it is better to record it than to ignore it. Do not assume
+"someone else" is handling an incident unless they have explicitly recorded it
+using the process defined here.
+
+## How do we _identify_ incidents?
+
+Incidents are detected through various channels such as automated alerts, user
+reports, or through manual checks by maintainers. The Mini Support Rotation
+(MSR) runner makes these checks as part of their daily responsibilities.
+
+While the MSR runner monitors our communication channels closely, **any online
+maintainer who sees something that might be an incident should follow the
+incident recording process outlined here.** This does not mean that the
+reporting maintainer is inherently responsible for stabilizing the incident or
+investigating the root cause. It only means that they are responsible for making
+the initial report on the incident. Additionally, maintainers who stabilize
+services may not be the ones to perform the investigation after stabilization
+has been reached. Every incident has a [**Lead**](#how-do-we-determine-the-lead)
+who is the person responsible for providing updates on the incident and
+delegating tasks. The Lead is also responsible for hosting the retrospective
+after the incident is resolved.
+
+## How do we _classify_ incidents?
+
+Incidents are classified in two ways:
+
+- **Severity** - The impact of the incident
+- **Status** - The current state of the incident
+
+### Severity
+
+Incidents are classified using one of three levels of severity. It is important
+to note that regardless of severity, working to stabilize services (or whatever
+harm mitigation is relevant for the incident) always **takes priority over
+regular project work and MSR duties.** Once services are stabilized, any ongoing
+investigations, preventative measures, or documentation changes will be
+prioritized by the team alongside our other work.
+
+This severity scale is intentionally distinct from our GitHub priority scale.
+Any GitHub issues created to _stabilize_ a Severity 1 or Severity 2 incident are
+marked as "critical" in GitHub. Other issues are prioritized at your own
+discretion.
+
+- **Severity 1**: Critical issue affecting all/most users and all/most
+  functionalities of Openverse.
+- **Severity 2**: Major issue affecting a large number of users or critical
+  functionality of Openverse.
+- **Severity 3**: Minor issue affecting a limited number of users or
+  non-critical functionality of Openverse.
+
+### Status
+
+- **Stabilization pending** - The incident is actively disrupting service.
+- **Stabilized** - The incident is no longer directly disrupting service. Root
+  causes may still be unknown.
+- **Under investigation** - The root causes of the incident are being
+  researched. Stabilized but not yet resolved.
```

Comment:
What's the difference between stabilized and under investigation? Is that to allow specific time after stabilisation where we haven't started looking into the root cause?

Reply:
I also had this question. Per the new text added below:

> Some incidents may not require further investigation once they are stabilized and can be immediately resolved.

But I think those incidents would go `Stabilization pending` straight to `Resolved`, so I'm still not sure when something would be `Stabilized` but not `Under investigation`. I guess it implies that investigation is needed but not started? Or, possibly, investigation has already happened and the cause is identified but not yet fixed? Although that can't be right, because that's the definition of `Resolved` given below.

Reply:
Just noticing that you commented on that for Resolved as well @sarayourfriend  [here](https://github.com/WordPress/openverse/pull/2679/files#r1421862675). _If_ Resolved is updated to mean that the long-term fixes are _implemented_, then one interpretation of the statuses which would maybe make sense  is:

* Stabilization pending: actively disrupting serice
* Under investigation: stabilized but not resolved. Root causes are being researched.
* Stabilized: Root causes identified but not yet fixed
* Resolved: Long-term fixes applied

But then the name `stabilized` feels like an inadequate description 😓

Reply:
Thanks for the further discussion here! I was trying to identify a state where things are no longer on fire, but perhaps we're not sure if things _need_ investigation or if they can be resolved. I honestly hadn't thought of moving straight from `Stabilization pending` to `Resolved` 😮 Perhaps that would cover the case I'm imagining well enough? Regardless, I'll make an issue for clarifying this!

Reply:
Issue created: https://github.com/WordPress/openverse/issues/3579